src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,15 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+        for (User user : users) {
+            if (email.equals(user.getEmail())) {
+                return user; // Возвращаем пользователя, если email совпал
+            }
+        }
+        return null; // Возвращаем null, если не нашли пользователя
```

Comment:
it's fine to have comments to make navigating the code for yourself easier, but please remove them before committing